src/FusedChannelData.ts

```diff
@@ -144,6 +144,7 @@ export default class FusedChannelData {
         colorRamp: {
           value: null,
         },
+        useRepeatingCategoricalColors: { value: false },
```

Comment:
This is functionality that was recently added to TFE 2D to support glasbey color ramps, which use a repeating color ramp with 256 colors. I've made the same changes here.